### PR TITLE
Issue 785 (Add "time since ticket logged" field )

### PR DESCRIPTION
**Pull request by saba83rish**
### Issue Reference
This PR addresses the Issue : Fixes # Issue 785 https://github.com/IEEEKeralaSection/rescuekerala/issues/785

### Summarize
Added a column for time elapsed 
Fixed a typo for name (in malayalam)

### CODE REVIEW

**Comment by saba83rish** [comment]
As I don't have all the data populated in my local please test this thoroughly. Or else give me a doc to populate some test data in my local.

**Comment by saba83rish** [comment]
image

**Comment by naishu** [comment]
I can test this, but im new

**Comment by saba83rish** [comment]
can we merge this please?

**Comment by saba83rish** [comment]
merged with latest master changes. pls merge

File: mainapp/templates/mainapp/request_list.html [hunk 1]
```diff
@@ -80,9 +80,10 @@ <h2 class="text-center">ഇതു വരെ ആവശ്യപ്പെട്ട
         <th>Request Number - അപേക്ഷ നമ്പര്‍</th>
         <th>District - ജില്ല</th>
         <th>Location - സ്ഥലം</th>
-        <th>Requestee - അപേക്ഷകന്‍റെ പേര</th>
+        <th>Requestee - അപേക്ഷകന്‍റെ പേര്</th>
         <th>Phone - ഫോണ്‍ നമ്പര്‍</th>
         <th>Date - തീയതി</th>
+        <th>Elapsed Time - അപേക്ഷ നൽകിയ സമയം </th>
```
Comment (biswaz) [inline mainapp/templates/mainapp/request_list.html]: The translation is not quite right. അപേക്ഷ നൽകിയ സമയം = Time of application right

Comment (saba83rish) [reply mainapp/templates/mainapp/request_list.html]: ok will change it. This was picked from the issue ticket.

Comment (saba83rish) [reply mainapp/templates/mainapp/request_list.html]: @vvazhayil let me know what would be the official wordings and i can change.

**Comment by vvazhayil** [comment]
I'd check with someone who can actually translate that. I don't claim to know it.

File: mainapp/templates/mainapp/request_list.html (hunk 1 above)
Comment (biswaz) [reply mainapp/templates/mainapp/request_list.html]: Make it : കഴിഞ്ഞു പോയ സമയം

**Comment by saba83rish** [comment]
@biswaz done. please merge

**Pull request merged**